WebView/ViewController.swift

```diff
@@ -6,14 +6,39 @@
 //
 
 import UIKit
+import WebKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, WKNavigationDelegate {
+    var webView: WKWebView!
+    
+    override func loadView() {
+        webView = WKWebView()
+        webView.navigationDelegate = self
+        view = webView
+    }
+        
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Open", style: .plain, target: self, action: #selector(openTapped))
+        guard let url = URL(string: "https://www.redspace.com") else { return }
+        webView.load(URLRequest(url: url))
+        webView.allowsBackForwardNavigationGestures = true
+    }
+    @objc func openTapped() {
+        let ac = UIAlertController(title: "Open page...", message: nil, preferredStyle: .actionSheet)
```

Comment:
Nitpick/minor: This is a matter of style, but my preference is to use more readable variable names, while trying to keep it not too long. So I'd go with `let alertController` it's a few extra characters but it's more expressive and readable in the code, and IDE helps you with code completion.